### PR TITLE
Feature/1 api

### PR DESCRIPTION
**Pull request by eddiejaoude**

### CODE REVIEW

File: src/DashboardHub/Bundle/DashboardBundle/Controller/DashboardController.php [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+namespace DashboardHub\Bundle\DashboardBundle\Controller;
+
+use DashboardHub\Bundle\DashboardBundle\Entity\Dashboard;
+use DashboardHub\Bundle\DashboardBundle\Form\DashboardType;
+use FOS\RestBundle\Controller\FOSRestController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Class YSlowStatsController
```
Comment (brettminnie) [inline src/DashboardHub/Bundle/DashboardBundle/Controller/DashboardController.php]: Wrong class name

File: src/DashboardHub/Bundle/DashboardBundle/Controller/DashboardController.php [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+ */
+class DashboardController extends FOSRestController
+{
+
+    /**
+     * @param Request $request
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function postDashboardAction(Request $request)
```
Comment (brettminnie) [inline src/DashboardHub/Bundle/DashboardBundle/Controller/DashboardController.php]: where does this post the dashboard to?

createDashboardAction probably is a better description

File: src/DashboardHub/Bundle/DashboardBundle/Controller/DashboardController.php [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+            $this->view($form, Response::HTTP_BAD_REQUEST)
+        );
+    }
+
+    /**
+     * @param Request $request
+     * @param string  $uid
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
```
Comment (brettminnie) [inline src/DashboardHub/Bundle/DashboardBundle/Controller/DashboardController.php]: I see you are using verbs, they don't help someone reading the code

File: src/DashboardHub/Bundle/DashboardBundle/Controller/DashboardController.php (hunk 1 above)
Comment (eddiejaoude) [inline src/DashboardHub/Bundle/DashboardBundle/Controller/DashboardController.php]: Good spot

File: src/DashboardHub/Bundle/DashboardBundle/Controller/DashboardController.php (hunk 2 above)
Comment (eddiejaoude) [inline src/DashboardHub/Bundle/DashboardBundle/Controller/DashboardController.php]: Has to follow FOSRestBundle convention http://symfony.com/doc/master/bundles/FOSRestBundle/3-listener-support.html

File: src/DashboardHub/Bundle/DashboardBundle/Controller/DashboardController.php (hunk 3 above)
Comment (eddiejaoude) [inline src/DashboardHub/Bundle/DashboardBundle/Controller/DashboardController.php]: See above

Comment (brettminnie) [inline src/DashboardHub/Bundle/DashboardBundle/Controller/DashboardController.php]: irritating, but ok

**Comment by brettminnie** [comment]
That was a bitch PR... mergereated

**Comment by eddiejaoude** [comment]
> That was a bitch PR... mergereated

Yes it was. Sorry! Try not to let it happen again.

**Pull request merged**